assets/js/base/components/price-slider/style.scss

```diff
@@ -149,6 +152,16 @@
 		margin-top: 0;
 		margin-left: 0;
 	}
+
+	.wc-block-price-filter__button {
+		&.show-loading-state {
+			@include placeholder();
```

Comment:
I see `@include placeholder()` causes `line-height` to be 1, so the button placeholder is a bit smaller than the real button. I think we could fix it setting `line-height: inherit;` to undo the placeholder line height. How does it sound?

Reply:
Good catch! It works with `line-heigh: initial`, `inherit` was making it bigger than the button.